### PR TITLE
escapting and echoing html in admin file

### PR DESCRIPTION
**Pull request by ramico**
Escaping and echoing html tags is a complicated process.

phpcs escaping of that is done via increasing level of cohesion in styling_checkbox_label method fixing 2 errors.

doc comment refliction of that is done

### CODE REVIEW

**Comment by ramico** [comment]
WordPress reference

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -156,14 +156,17 @@ private function output_settings_main() {
...
+		 * Echoing the style rule for the next sibbling of checkbox label to display inline
 		 *
-		 * @param string $label_for The ID of the field.
-		 *
-		 * @return string The outputted HTML
+		 * @param string $label_for same of The ID of the checkbox html validation.
 		 */
 		private function styling_checkbox_label( $label_for ) {
-			return sprintf( '<style type="text/css">label[for="%1$s"]+p{display:inline}</style>', $label_for );
+			$allowed_html =
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:164]: I think we can make this a class constant.

File: includes/class-wc-siftscience-admin.php [hunk 2]
```diff
@@ -156,14 +160,12 @@ private function output_settings_main() {
...
-		 *
-		 * @param string $label_for The ID of the field.
+		 * Echoing the style rule for the next sibbling of checkbox label to display inline
 		 *
-		 * @return string The outputted HTML
+		 * @param string $label_for same of The ID of the checkbox html validation.
 		 */
 		private function styling_checkbox_label( $label_for ) {
-			return sprintf( '<style type="text/css">label[for="%1$s"]+p{display:inline}</style>', $label_for );
+			echo wp_kses( sprintf( '<style type="text/css">label[for="%1$s"]+p{display:inline}</style>', $label_for ), self::ALLOWED_HTML );
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:168]: This line is quite long. I think it would look better if you split it into two statements:

```
$html = sprintf(....)
echo wp_kses( $html.....)
```

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:168]: your right but come to think of it no need to 

as you like

File: includes/class-wc-siftscience-admin.php [hunk 3]
```diff
@@ -141,29 +145,36 @@ public function output_settings_fields() {
...
 		 */
 		private function output_settings_main() {
 			WC_Admin_Settings::output_fields( $this->get_settings() );
 
-			echo $this->styling_checkbox_label( WC_SiftScience_Options::AUTO_SEND_ENABLED );
-			echo $this->batch_upload();
-			$data = array( 'api' => admin_url( 'admin-ajax.php' ) );
+			$this->styling_checkbox_label( WC_SiftScience_Options::AUTO_SEND_ENABLED );
+
+			echo <<<'table'
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:155]: Interesting. Why did you move the the code in this case, but not for `styling_checkbox_label`?

Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:155]: Also, this needs `wp_kses` :-)

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:155]: label code was used twice unlike batch control

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:155]: strangely I was not prompt

**Comment by ramico** [comment]
output should only be escaped IF it contains user defined variables - my findings
